src/test/java/com/cloudbees/jenkins/support/util/Chrono.java

```diff
@@ -0,0 +1,225 @@
+package com.cloudbees.jenkins.support.util;
+
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static java.util.concurrent.TimeUnit.DAYS;
+import static java.util.concurrent.TimeUnit.HOURS;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class Chrono {
```

Comment:
A class used for performance tests, used only in test environment.

Reply:
Where is the class used?

Reply:
I used this to measure inside the _writebundle_ method and other places what is taking the most of the time. I left it there for future use, as it's in test scope it doesn't go to production.